### DISCUSSION TITLE
如何设置摘要为空，且不独占一行？

### DISCUSSION THREAD

**Discussion by GanZhiXiong** [ppoffice/hexo-theme-icarus#959] [Q&A] [answered]
以下下情况都会有摘要，摘要内容为 `<!--more-->` 后面的内容：
- `<!--more-->` 前面没有任何内容
- `<!--more-->` 上一行有 1 个或 2 个 或 3 个空格

但是当 `<!--more-->` 上一行有 4 个空格时，摘要内容为空，但摘要独占一行，这样就有点占页面空间了，而且空白的一行不美观，所以有没有办法使摘要为空，而且不独占一行？

![image](https://user-images.githubusercontent.com/27394631/139104756-d6dd9e44-4884-4872-ab6a-ac8436e3eaa3.png)

![image](https://user-images.githubusercontent.com/27394631/139104797-c0e56657-d37b-42e9-be9b-8aad83bd8874.png)

**Comment by ppoffice** [comment, marked as answer]
@GanZhiXiong

```diff
  title: Your post title
  ---
+ <div></div>
+ <!--more-->
  Rest of your article
```

**Reply by GanZhiXiong** [reply]
非常感谢您的回复！经测试能解决此问题！
